microsoft-edge/devtools-guide-chromium/memory-problems/memory-101.md

```diff
@@ -23,216 +23,166 @@ ms.date: 07/21/2023
    limitations under the License. -->
 # Memory terminology
 
-This article describes common terms used in memory analysis, and is applicable to various memory profiling tools for different languages.
+This article describes common terms used in the **Memory** tool, a tool for investigating memory issues, but is applicable to memory analysis in general and to various memory profiling tools for different languages. If you've worked with memory profilers for languages such as Java or .NET, this article may be a refresher.
```

Comment:
```suggestion
The following terms are used in the **Memory** tool, which is for investigating memory issues.  These memory terms are applicable to memory analysis in general, as well as memory profiling tools for languages such as Java or .NET.
```
@captainbrosset - did this interesting rewrite of important Paragraph 1 fyi.
* Talk in terms of tech, not docs.  Keep focus on tech, not docs.
* Break up long sentence, as Acrolinx probably says.
* Uncomfortable talking about the reader's level of knowledge; not necessary.  Keep focus on tech, not speculating about user's level of knowledge.